Junimatic/ObjectMachine.cs

```diff
@@ -135,6 +135,14 @@ public override bool FillMachineFromChest(GameStorage storage, Func<Item, bool>
                 return false;
             }
 
+            // Don't attempt to load items into machines that don't accept any input
+            // Check for is_machine to avoid breaking crab pots
+            if (this.Machine.HasContextTag("is_machine") && !this.Machine.HasContextTag("machine_input"))
```

Comment:
I wonder if we could do this one better by caching this discovery.  (My quick scan of HasContextTag's implementation suggests it's in the cheap-but-not-free level of call costs)

Reply:
Possibly - I'm not familiar enough with the system to know if anything would change these on the fly though.

Reply:
Yeah, we should hold onto that idea.  There are more things that are cacheable like that, better to do it at once.  I skipped doing a bunch of optimizations I thought of and just left in a limited form of instrumentation (in the form of a few trace statements with timers).